docs/cms-query-integration.md

```diff
@@ -0,0 +1,68 @@
+# CMS Query Integration
+
+The Search module overrides the default search functionality to query the specialized search index in place of a standard MySQL query. This means all default search operations using the CMS search APIs for posts, users and terms such as `WP_Query`, `WP_User_Query`, `WP_Term_Query` and the REST API search endpoints will transparently make use of the search index by default.
+
+It is possible to control this behaviour and thus use the search index for non search queries that might otherwise be too slow to do in the database such as meta queries.
```

Comment:
"non search" should be hyphenated, and there should be commas.

```suggestion
It is possible to control this behaviour, and thus use the search index for non-search queries that might otherwise be too slow to do in the database, such as meta queries.
```

Reply:
The second comma after database reads a bit funny to me, suggesting this instead:
```suggestion
It is possible to control this behaviour and thus use the search index for non-search queries. The reason for doing this is to improve performance when running complex queries, such as meta queries, that would otherwise be too slow to do in the database.
```

Reply:
LGTM